app/definitions/index.ts

```diff
@@ -0,0 +1,10 @@
+import { StackNavigationProp } from '@react-navigation/stack';
+import { Dispatch } from 'redux';
+
+export interface BaseScreen {
+	navigation: StackNavigationProp<any>;
```

Comment:
As discussed yesterday, did you check if we'll lost the navigation type already implemented ? 
and if is possible pass a generic type for this navigation?
```
export interface BaseScreen<T, S> {
    navigation: StackNavigationProp<T, S>
```